packages/rollup-plugin/src/index.ts

```diff
@@ -45,6 +45,12 @@ function rollupDebugIdUploadPlugin(
   logger: Logger,
   createDependencyOnBuildArtifacts: () => () => void
 ): UnpluginOptions {
+  // Only for testing purposes
+  if (process.env["SENTRY_NODE_ENV"] === "test") {
```

Comment:
I think for rollup and vite, we could simply assert on the names of the returned plugins in some unit tests. For webpack, we probably need something like this indeed 🤔 so no strong objections to not doing it